transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -360,23 +360,13 @@ public String toString() {
         SocketAddress remoteAddr = remoteAddress();
         SocketAddress localAddr = localAddress();
         if (remoteAddr != null) {
-            SocketAddress srcAddr;
-            SocketAddress dstAddr;
-            if (parent == null) {
-                srcAddr = localAddr;
-                dstAddr = remoteAddr;
-            } else {
-                srcAddr = remoteAddr;
-                dstAddr = localAddr;
-            }
-
             StringBuilder buf = new StringBuilder(96)
                 .append("[id: 0x")
                 .append(id.asShortText())
                 .append(", ")
-                .append(srcAddr)
-                .append(active? " => " : " :> ")
-                .append(dstAddr)
+                .append(localAddr)
+                .append(active? " - " : " ! ")
```

Comment:
ok, maybe different people like different sytle so we can choose this way.NP, I will update the commit codes. 3ks.